app_modules/c3Helper.js

```diff
@@ -15,9 +15,10 @@ function createDateLabels() {
   var dateObj = new Date();
 
   for (i = 0; i < 6; i++) {
+    var year = dateObj.getFullYear() - 2000;
```

Comment:
Are you doing this to get just the last two digits of the year? Doesn't the chart's `xFormat` property already handle that?